internal/app/tfsec/config/config.go

```diff
@@ -0,0 +1,51 @@
+package config
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/tfsec/tfsec/internal/app/tfsec/debug"
+	"gopkg.in/yaml.v2"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+type Config struct {
+	SeverityOverrides map[string]string `json:"severity_overrides,omitempty" yaml:"severity_overrides,omitempty"`
+}
+
+func LoadConfig(configFilePath string) (*Config, error) {
+	var config = &Config{}
+
+	if _, err := os.Stat(configFilePath); err != nil {
+		if os.IsNotExist(err) {
+			debug.Log("Failed to load the config file, not found. %s", configFilePath)
+			return config, nil
+		}
+	}
+
+	configFileContent, err := ioutil.ReadFile(configFilePath)
+	if err != nil {
+		return config, err
```

Comment:
Normally I prefer return nil when an error is returned, but I guess in this case we want a default empty config?